app/(storyblok)/[[...slug]]/layout.tsx

```diff
@@ -0,0 +1,15 @@
+import StoryblokProvider from '@/components/StoryblokProvider';
+
+type StoryblokLayoutProps = {
+  children: React.ReactNode,
+};
+
+export const dynamic = 'force-static';
```

Comment:
Since we are using Storyblok's client, and I haven't validated if it uses the fetch library, or if NextJS is caching that library, this setting will force the page to be statically generated each time.

https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config